quotes/facts.txt

```diff
@@ -15,3 +15,74 @@ China produces the world's second largest crop of soybeans.
 William Shakespeare did not exist. His play were masterminded in 1589 by Francis Bacon, who used an Ouija board to enslave play-writing ghosts.
 Error. Error. Error. Fact not found.
 At some point in their lives 1 in 6 children will be abducted by the dutch.
+
```

Comment:
I don't think the `quotes.py` script can handle empty lines. Consider adding `list(filter(None, content.splitlines()))`

Reply:
Resolved with #23